py_zipkin/zipkin.py

```diff
@@ -709,3 +713,125 @@ def create_http_headers_for_new_span(context_stack=None, tracer=None):
         'X-B3-Flags': '0',
         'X-B3-Sampled': '1' if zipkin_attrs.is_sampled else '0',
     }
+
+
```

Comment:
This file is getting very long and these functions don't really depend on anything else defined in here. What about creating a new file, something like request_helpers.py or so

Reply:
Will do!  I'm wrangling the structure around, then I'll start in on all the other (great!) suggestions.